### ISSUE TITLE
Rounding total price goes wrong

### ISSUE THREAD

**Issue by ramon-bakker** [kimai/kimai#480]
I found a bug with rounding total prices.
Example:
- Price: 15.00
- Duration (h:m): 3:28

3:28 converts to 3+1/60x28 = 3.46666... and the report shows 3.47 hours.
The total price should then be 3.47x15.00 = 52.05. But it takes the exact value of 3+1/60x28 and then calculates the total price, which gives 52.00.

The result is not correct. It would be nice to have an option in the Kimai settings where you can set a rounding preference. The already existing option 'Use exact sums instead of adding the shown rounded values.' does not work for this. It tried to enable and disable the option, but it gives me always 52.00.

**Comment by kevinpapst** [comment]
Which place are you talking about? Invoice? Time table?

**Comment by ramon-bakker** [comment]
It applies to the time table.

**Comment by ramon-bakker** [comment]
I have fixed it in my fork.
![db_layer_mysql](https://cloud.githubusercontent.com/assets/10447087/8144705/0835fcaa-11eb-11e5-8c6c-0e51f80590f9.jpg)
![db_layer_pdo](https://cloud.githubusercontent.com/assets/10447087/8144707/0b18e87e-11eb-11e5-90ee-758784b1c25a.png)
![private_func](https://cloud.githubusercontent.com/assets/10447087/8144708/0c92c21a-11eb-11e5-97ca-1183245ad324.jpg)

**Comment by kevinpapst** [comment]
Cool, then why not sent in a pull request?!

**Comment by ramon-bakker** [comment]
I can not do a pull request, because my forked repository is not hosted on GitHub.

**Comment by kevinpapst** [comment]
And what about a patch? That should be possible, even with an "outside" repo :)

**Comment by ramon-bakker** [comment]
I have created a patch: #23b4f10d0caaf9430358

**Comment by kevinpapst** [comment]
Cool! We will incorporate this after the upcoming release.

**Comment by kevinpapst** [comment]
Why closed?

**Comment by ramon-bakker** [comment]
Sorry, I didn't know that I must not close it if it is not yet in the release. I thought I could close it, because I provided the patch.

**Comment by kevinpapst** [comment]
I will close it, once it is merged :-) Thanks for your effort!!!

2015-06-19 15:24 GMT+02:00 Ramon Bakker <notifications@github.com>:

**Comment by ramon-bakker** [comment]
Ok, you're welcome. Thanks for your help.

**Comment by kevinpapst** [comment]
Kimai 1 is not maintained any longer.
This issue will now be closed.
Please [read more about Kimai 2 here](https://github.com/kimai/kimai/issues/1291).
